### PR TITLE
Add Rotary improvements for Picker snap scroll

### PR DESCRIPTION
**Pull request by Kpeved**
#### WHAT
Add Rotary improvements for Picker snap scroll

#### WHY
Rotary works better with new parameters

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: composables/src/main/java/com/google/android/horologist/composables/picker/PickerRotaryScrollAdapter.kt [hunk 1]
```diff
@@ -32,11 +32,11 @@ internal class PickerRotaryScrollAdapter(
 ) : RotaryScrollAdapter {
 
     /**
-     * Returns a height of a centered item, as all items in picker have the same height.
+     * Returns a height of a first item, as all items in picker have the same height.
      */
     override fun averageItemSize(): Float =
         scrollableState.scalingLazyListState
-            .layoutInfo.visibleItemsInfo[currentItemIndex()].size.toFloat()
+            .layoutInfo.visibleItemsInfo[0].unadjustedSize.toFloat()
```
Comment (luizgrp) [inline composables/src/main/java/com/google/android/horologist/composables/picker/PickerRotaryScrollAdapter.kt:39]: nit: .first

File: compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/Rotary.kt [hunk 2]
```diff
@@ -56,9 +59,10 @@ import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.flow.receiveAsFlow
 import kotlinx.coroutines.flow.transformLatest
 import kotlin.math.abs
+import kotlin.math.absoluteValue
 import kotlin.math.sign
 
-private const val DEBUG = false
+private const val DEBUG = true
```
Comment (luizgrp) [inline compose-layout/src/main/java/com/google/android/horologist/compose/rotaryinput/Rotary.kt]: I think you might want to revert this

File: media/ui/api/current.api [hunk 3]
```diff
@@ -44,6 +44,10 @@ package com.google.android.horologist.media.ui.components {
     method @androidx.compose.runtime.Composable public static void ControlButtonLayout(kotlin.jvm.functions.Function0<kotlin.Unit> leftButton, kotlin.jvm.functions.Function0<kotlin.Unit> middleButton, kotlin.jvm.functions.Function0<kotlin.Unit> rightButton, optional androidx.compose.ui.Modifier modifier);
   }
 
+  public final class CustomActionMediaButtonKt {
```
Comment (luizgrp) [inline media/ui/api/current.api:47]: 😱 thanks for updating this, I wonder why it didn't fail in CI before

Comment (Kpeved) [reply media/ui/api/current.api:47]: ¯\_(ツ)_/¯

**Review by luizgrp — APPROVED** [review]

**Pull request merged**